cmd/root.go

```diff
@@ -12,48 +12,58 @@ import (
 )
 
 var (
-	cfgFile    string
+	cfgKey     string
 	OtdfctlCfg config.Config
 
 	configFlagOverrides = config.ConfigFlagOverrides{}
 )
 
-// rootCmd represents the base command when called without any subcommands
+// RootCmd represents the base command when called without any subcommands.
 var (
-	rootCmd = &man.Docs.GetDoc("<root>").Command
+	RootCmd = &man.Docs.GetDoc("<root>").Command
 )
 
 func init() {
 	doc := man.Docs.GetDoc("<root>")
-	rootCmd = &doc.Command
-	rootCmd.PersistentFlags().String(
+	RootCmd = &doc.Command
+	RootCmd.PersistentFlags().String(
 		doc.GetDocFlag("host").Name,
 		doc.GetDocFlag("host").Default,
 		doc.GetDocFlag("host").Description,
 	)
-	rootCmd.PersistentFlags().StringVar(
-		&cfgFile,
-		doc.GetDocFlag("config-file").Name,
-		doc.GetDocFlag("config-file").Default,
-		doc.GetDocFlag("config-file").Description,
-	)
-	rootCmd.PersistentFlags().String(
+	RootCmd.PersistentFlags().String(
 		doc.GetDocFlag("log-level").Name,
 		doc.GetDocFlag("log-level").Default,
 		doc.GetDocFlag("log-level").Description,
 	)
+}
 
-	cfg, err := config.LoadConfig("otdfctl")
+// Execute adds all child commands to the root command and sets flags appropriately.
+// The config file and key are defaulted to otdfctl.yaml.
+func Execute() {
+	cfg, err := config.LoadConfig("", "otdfctl")
 	if err != nil {
 		fmt.Println("Error loading config:", err)
 		os.Exit(1)
 	}
 	OtdfctlCfg = *cfg
+	err = RootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
```

Comment:
```suggestion
	ExecuteWithBootstrap("", "otdfctl")
```

Reply:
I tested this and we actually want the key argument as "" as well. Pushing up.